apple/internal/platform_support.bzl

```diff
@@ -35,6 +35,7 @@ _DEVICE_FAMILY_VALUES = {
     "ipad": 2,
     "tv": 3,
     "watch": 4,
+    "vision": 7,
```

Comment:
Not sure if this used to be `vision`, but it is `reality` in beta 4

```suggestion
    "reality": 7,
```

Reply:
From B4

<img width="204" alt="CleanShot 2023-07-13 at 22 42 37@2x" src="https://github.com/bazelbuild/rules_apple/assets/75236/69fbd800-a402-4675-8d28-e963fdf827e8">

Reply:
fixed, thanks